front/src/trackview/Trackview.js

```diff
@@ -137,7 +137,9 @@ class Trackview extends Component {
   }
 
   backlink() {
-    return `/dayview/${this.props.match.params.date}`;
+    const date = new Date(this.state.date.setDate(this.state.date.getDate()));
```

Comment:
Is the setDate-getDate necessary for some bugfix? It's otherwise redundant

new Date(this.state.date.getDate());

Reply:
I dont remember this change, but if removed returning from trackview to dayview send to current date instead of the date returned from

Reply:
Okay, let's keep it that way.